web/src/main/webapp/WEB-INF/config-db/jdbc.properties

```diff
@@ -1,6 +1,6 @@
-jdbc.username=admin
-jdbc.password=gnos
-jdbc.database=geonetwork
+jdbc.username=ngr
```

Comment:
This shouldn't be here, you probably want to remove this pull request before some  hacker sees it ;)